features/src/main/feature/camel-features.xml

```diff
@@ -1403,6 +1403,28 @@
         <bundle dependency='true'>mvn:io.netty/netty-codec-http/4.1.107.Final</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-olingo4/${project.version}</bundle>
     </feature>
+    <feature name='camel-openstack' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jackson</feature>
+        <feature version='${guava-version}'>guava</feature>
+        <!-- use wrap to override the guava version -->
+        <bundle dependency='true'>wrap:mvn:com.github.openstack4j.core/openstack4j-core/${openstack4j-version}$overwrite=merge&amp;Import-Package=com.fasterxml*,com.github*,javax*,org*,com.google.common.base;version=33.0.0</bundle>
+        <bundle dependency='true'>wrap:mvn:com.github.openstack4j.core.connectors/openstack4j-okhttp/${openstack4j-version}$overwrite=merge&amp;Import-Package=com.fasterxml*,okhttp3*,javax*,org*;resolution:=optional,com.google.common.base;version=33.0.0</bundle>
+        <bundle dependency='true'>wrap:mvn:com.github.java-json-tools/json-patch/${json-patch-version}$overwrite=merge&amp;Export-Package=com.github.fge*;version=${json-patch-version}&amp;Import-Package=com.fasterxml*,com.github;resolution:=optional;javax*</bundle>
```

Comment:
according to my `mvn dpendency:tree` and https://central.sonatype.com/artifact/com.github.openstack4j.core/openstack4j-parent/3.11, json-patch-version should be 1.9 (with old group id `om.github.fge`), but I think that we should push for 1.13, `${json-patch-versio}`

Reply:
Import-Package is needed to override `javax.annotation;version="[3.0,4)"` (doesn't exists) by using `javax*`.

Reply:
Export-Package is needed because the original bundle doesn't exports anything

Reply:
❓ Suggestion to make it clear why the 4 bundles below are needed
```
<bundle dependency='true'>wrap:mvn:com.github.java-json-tools/json-patch/${json-patch-version}$overwrite=merge&amp;Export-Package=com.github.fge*;version=${json-patch-version}&amp;Import-Package=com.fasterxml*,com.github*;javax*</bundle>
<bundle dependency='true'>wrap:mvn:com.github.java-json-tools/jackson-coreutils/2.0$overwrite=merge&amp;Export-Package=com.github.fge*;version=2.0&amp;Import-Package=com.fasterxml*,com.github*;javax*</bundle>
<bundle dependency='true'>mvn:com.github.java-json-tools/msg-simple/1.2</bundle>
<bundle dependency='true'>mvn:com.github.java-json-tools/btf/1.3</bundle>
```

Reply:
good catch for the inconsistencies of groupId !